code/modules/reagents/reagent_containers/powderspice.dm

```diff
@@ -186,7 +186,6 @@
 	volume = 15
 	list_reagents = list(/datum/reagent/ozium = 15)
 	sellprice = 5
-	possible_potion = "perpot"
```

Comment:
why do this?

Reply:
Why remove the "possible potion" var? Its entirely unused now, as all alchemy reagents have been moved under the subtype of "obj/item/alch" with the "Alchemical Grinder" being used to turn things into "alchemically ready" substances
Check out ingredients.dm under roguecrafting/alchemy